extras/upgrade_activities.sh

```diff
@@ -28,7 +29,13 @@ function my_exit()
   # kill gluster-blockd
   if [[ ${GBD_STARTED} -ne 0 ]]
   then
-    pkill -15 gluster-blockd
+    kill -15 ${GBD_PID} > /dev/null 2>&1
```

Comment:
What's the point in killing if GBD_PID=0 ?

lets check for ${GBD_STARTED} -ne 0 && ${GBD_PID} -ne 0 ? what do you think ?

Reply:
In theory the GBD_PID shouldn't be 0.
But agree to do the check here.